dipy/tracking/utils.py

```diff
@@ -954,8 +958,13 @@ def move_streamlines(streamlines, output_space, input_space=None):
     yield
     # End of initialization
 
-    for sl in streamlines:
-        yield np.dot(sl, lin_T) + offset
+    if return_seeds:
+        # Seeds are already moved
```

Comment:
This comment is confusing: are they already moved? Why are you applying the transformation to them?

Reply:
Woops, yes at first I thought they would already be in the same space, but when I rechecked with another script they weren't. The transformation does need to be applied so that seeds and streamlines are in the same space. I'll change the comment

Reply:
Actually I'll add a test just to make sure they're in the same space, too.

Reply:
Should be fixed in 07e9d79 !